README_JP6.md

```diff
@@ -95,61 +96,40 @@ Note: dev_dbg() log support will not be enabled by default. If needed, run the `
-echo "Archiving kernel HID modules"
-tar -czf ./images/6.2/kernel-hid-modules.tar.gz -C "./images/6.2/rootfs/" ./lib/modules/5.15.148-tegra/extra
-```
 ## Backup JetPack 6.2 boot configuration and drivers (optional)
 ```
 echo "Backup boot configuration"
 sudo cp /boot/tegra234-p3737-0000+p3701-0000-nv.dtb /boot/tegra234-p3737-0000+p3701-0000-nv-bkp.dtb
-
 # Note: If using a production board and not a dev kit copy the relevant dtb file below
 sudo cp /boot/tegra234-p3737-0000+p3701-0005-nv.dtb /boot/tegra234-p3737-0000+p3701-0005-nv-bkp.dtb
-
-echo "backup nvidia-oot modules"
-sudo tar -czf /lib/modules/5.15.148-tegra/updates.tar.gz -C /lib/modules/5.15.148-tegra/ updates
 ```
 
 ## Install kernel drivers, extra modules and device-tree to Jetson AGX Orin
 
 Following steps required:
 
-1.	Copy entire directory `images/6.2/rootfs/lib/modules/5.15.148-tegra/updates` from host to `/lib/modules/5.15.148-tegra/` on Orin target
-2.	Copy entire directory `images/6.2/rootfs/lib/modules/5.15.148-tegra/extra` from host to `/lib/modules/5.15.148-tegra/` on Orin target
-3.	Copy `tegra234-camera-d4xx-overlay.dtbo` from host to `/boot/tegra234-camera-d4xx-overlay.dtbo` on Orin target
-4.	For dual camera, copy `tegra234-camera-d4xx-overlay-dual.dtbo` from host to `/boot/tegra234-camera-d4xx-overlay-dual.dtbo` on Orin target
-5.  Copy `tegra234-p3737-0000+p3701-0000-nv.dtb` from host to `/boot/` on Orin
-6.  Copy `Image` from host to `/boot/` on Orin
-7.  Copy entire directory `images/6.2/rootfs/lib/modules/5.15.148-tegra/kernel` from host to `/lib/modules/5.15.148-tegra/` on Orin target
-8.	Run  $ `sudo /opt/nvidia/jetson-io/jetson-io.py`
-    1.	Configure Jetson AGX CSI Connector
-    2.	Configure for compatible hardware
-    3.	Choose appropriate configuration:
-        1. Jetson RealSense Camera D457
-        2. Jetson RealSense Camera D457 dual
-    4.  Enable depmod scan for "extra" & "kernel" modules $ `sudo sed -i 's/search updates/search extra updates kernel/g' /etc/depmod.d/ubuntu.conf`
-        ```
-        $ cat /etc/depmod.d/ubuntu.conf
-        search extra updates kernel ubuntu built-in
-        ```
-    5.	$ `sudo depmod`
-    6.	$ `echo "d4xx" | sudo tee /etc/modules-load.d/d4xx.conf`
-    
-9.  Verify bootloader configuration
+1.	Copy entire directory `images/6.2/rootfs/lib/modules/5.15.148-tegra` from host to `/lib/modules/` on Orin target
+2.	Copy `tegra234-camera-d4xx-overlay.dtbo` from host to `/boot/tegra234-camera-d4xx-overlay.dtbo` on Orin target
+3.	For dual camera, copy `tegra234-camera-d4xx-overlay-dual.dtbo` from host to `/boot/tegra234-camera-d4xx-overlay-dual.dtbo` on Orin target
+4.	Copy `tegra234-p3737-0000+p3701-0000-nv.dtb` from host to `/boot/` on Orin
+5.	Copy `Image` from host to `/boot/` on Orin
+6.	Run  $ `sudo /opt/nvidia/jetson-io/jetson-io.py`
+	1.	Configure Jetson AGX CSI Connector
+	2.	Configure for compatible hardware
+	3.	Choose appropriate configuration:
+		1. Jetson RealSense Camera D457
```

Comment:
We need to think if this is all MIPI or just D457, not for now..
But we have D430 GMSL and D415 GMSL